### DISCUSSION TITLE
Suggestion: tell user about reinstalling IPAs for older incompatible app installation

### DISCUSSION THREAD

**Discussion by RigoLigoRLC** [PlayCover/PlayCover#128] [General]
I had upgraded from 1.0.4 to 1.0.7 today, and my apps disappeared from the list. After a short conversation on Discord, I was told the older versions of PlayCover has architectural differences, and thus automatic migration is impossible.

I was thinking that PlayCover should warn the user about incompatible older installations and make the user reinstall them, rather than leaving them out and gives no errors. Making no noise is the worst situation IMO. It could be unnecessary for the current version, but it would be appreciated if next breaking update gets this feature.

**Comment by ZhichGaming** [comment]
Also the pop-up should give them the option to delete them. Also needs to tell them that app data will be conserved, like the 12gb Genshin download.

**Comment by JoseMoreville** [comment]
I'm going to have that noted, but for a good practice try always reinstalling the app you want to use after you update playcover
